src/config/types.ts

```diff
@@ -440,6 +444,9 @@ export class PointDataLayerProps extends CommonLayerProps {
 
   @optional
   boundary?: LayerKey;
+
+  @optional
+  processing?: PointDataProcessing;
```

Comment:
is it `processing` or `loading`?

Reply:
I'm a bit confused about this `processing` enum and what it's used for? Can you explain?